src/xtopcom/xvm/xsystem_contracts/src/xzec_slash_info_contract.cpp

```diff
@@ -544,7 +578,14 @@ xunqualified_node_info_t xzec_slash_info_contract::process_statistic_data(top::d
                     xdbg("[xzec_slash_info_contract][do_unqualified_node_slash] incremental validator data: {gourp id: %d, account addr: %s, slot id: %u, subset count: %u, block_count: %u}", group_addr.group_id().value(), account_addr.c_str(),
                         slotid, group_account_data.account_statistics_data[slotid].vote_data.block_count, group_account_data.account_statistics_data[slotid].vote_data.vote_count);
                 }
-
+            } else if (top::common::has<top::common::xnode_type_t::evm>(group_addr.type())) {// process evm group
+                for (std::size_t slotid = 0; slotid < group_account_data.account_statistics_data.size(); ++slotid) {
+                    auto account_addr = node_service->get_group(group_xvip2)->get_node(slotid)->get_account();
+                    res_node_info.evm_info[common::xnode_id_t{account_addr}].subset_count += group_account_data.account_statistics_data[slotid].vote_data.block_count;
```

Comment:
double check to see if this API(process_statistic_data) is still used somewhere in our code base. i think this API should be removed since it uses node_service which is not allowed and is changed already.